packages/0x.js/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## v0.32.2 - _February 9, 2018_
```

Comment:
These new CHANGELOG entries aren't published just yet so they should technically be labeled as TBD but if we're publishing right after this, should be ok

Reply:
We are.